Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -677,25 +683,23 @@ CertReader::CertReader(std::vector<u8>&& bytes) : SignedBlobReader(std::move(byt
   if (!IsSignatureValid())
     return;
 
-  switch (GetSignatureType())
-  {
-  case SignatureType::RSA4096:
-    if (m_bytes.size() < sizeof(CertRSA4096))
-      return;
-    m_bytes.resize(sizeof(CertRSA4096));
-    break;
+  static constexpr std::array<std::tuple<SignatureType, PublicKeyType, size_t>, 4> TYPES{{
```

Comment:
Does this one really have to scream like that?

Reply:
I think so, the contributing guide says 'all compile time constants should be fully uppercased'. The name is short enough to not become too hard to read, hopefully :P

Reply:
I think we should consider removing that rule, and just use regular casing rules instead of special-casing like that. The only thing we probably want upper-cased is macros only (most of the guide was written before C++11, with additions being added over time, iirc)